### DISCUSSION TITLE
[Question] How to use Kernel Explainer when there are categorical features?

### DISCUSSION THREAD

**Discussion by mohammad-saber** [shap/shap#2222] [Q&A]
Hi, thank you for reading my question. 

Thank you for reading my question. I would like to know if I can use Kernel Explainer when there are categorical features in dataset. Problem is regression, not classification.

**Comment by sergiomujica** [comment]
you will need to one hot encode your categorical feature. You can also use an embedding. If your model is a neural network in Tensorflow you can do it using keras preprocessing layers.
